server/request_processor.go

```diff
@@ -163,7 +168,9 @@ func (r *RpcRequest) sendTxToRelay() {
 	txHash := strings.ToLower(r.tx.Hash().Hex())
 
 	// Check if tx was already forwarded and should be blocked now
-	if r.blockResendingTxToRelay(txHash) {
+	wasSentToRelay := r.blockResendingTxToRelay(txHash)
+	r.ethSendRawTxEntry.WasSentToRelay = wasSentToRelay
```

Comment:
here, `ShouldSendToRelay` should be set to true

Reply:
That means `NeedsFrontRunningProtection = ShouldSendToRelay ` ?

Reply:
oh right, we have that already! good point. could remove `ShouldSendToRelay`? 🤔

Reply:
Removed `ShouldSendToRelay`